src/app/projects/states/dashboard/directives/task-dashboard/directives/task-submission-card/task-submission-card.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, Input, Inject, OnInit } from '@angular/core';
+import { gradeService, Task, taskService } from 'src/app/ajs-upgraded-providers';
+
+@Component({
+  selector: 'task-submission-card',
+  templateUrl: 'task-submission-card.component.html',
+  styleUrls: ['task-submission-card.component.scss'],
+})
+export class TaskSubmissionnCardComponent implements OnInit {
+  @Input() task: any;
+
+  submission_date: String;
+  public canReuploadEvidence: any;
+  public canRegeneratePdf: any;
```

Comment:
Are these really "any"? Why not boolean?

Reply:
These re code changes done.